model/vm_host.rb

```diff
@@ -196,6 +196,15 @@ def download_firmware(version_x64: nil, version_arm64: nil, sha256_x64: nil, sha
     Strand.create_with_id(schedule: Time.now, prog: "DownloadFirmware", label: "start", stack: [{subject_id: id, version: version, sha256: sha256}])
   end
 
+  # Introduced for downloading cloud hypervisor via REPL.
+  def download_cloud_hypervisor(version_x64: nil, version_arm64: nil, sha256_ch_bin_x64: nil, sha256_ch_bin_arm64: nil, sha256_ch_remote_x64: nil, sha256_ch_remote_arm64: nil)
+    version, sha256_ch_bin, sha256_ch_remote = (arch == "x64") ? [version_x64, sha256_ch_bin_x64, sha256_ch_remote_x64] : [version_arm64, sha256_ch_bin_arm64, sha256_ch_remote_arm64]
```

Comment:
So uh, can we have some defaulting here? It's not like the artifacts ought to change after their release, we can build a little database in a hash literal here, right?

Reply:
Added defaulting to the Prog. You can now use `download_cloud_hypervisor(version_x64: "35.1")` (for an x64 host)

Reply:
...oof, but if you typo, it assumes things are arm, because its not x64? Can it be better still?

Reply:
`download_cloud_hypervisor(version_x64: "35.1")` works for x64 but fails for arm with ArgumentError.
`download_cloud_hypervisor(version_arm64: "35.2")` works for arm but fails for x64 with ArgumentError.
`download_cloud_hypervisor(version_x64: "35.1, version_arm64: "35.2")` works for both and can be used to download to all hosts in the fleet.

Reply:
I feel it's good that `download_cloud_hypervisor(version_x64: "35.1")` fails on an arm host but you can still use a single command like `VmHost.all.map { _1.download_cloud_hypervisor(version_x64: "35.1, version_arm64: "35.2") }` to download the appropriate version to the whole fleet.

Reply:
I think we have the assumption that arch is either `x64` or `arm64` all over the place, but I'm happy to extend this check.

Reply:
do we? This is kinda why I made `Arch.render`, to avoid this kind of thing

Reply:
updated this to fail for unexpected architectures. 

we do this check in `LearnArch`: `fail "BUG: unexpected CPU architecture" unless ["arm64", "x64"].include?(arch)`, which provides some guarantees about valid architectures. I think we do have a couple of places in the code where we check for x64/arm64 and if it's not that one we assume it's the other. but I agree that's not optimal.